src/detectors/FHCAL/ProtoCluster_factory_LFHCALIslandProtoClusters.h

```diff
@@ -30,8 +34,8 @@ class ProtoCluster_factory_LFHCALIslandProtoClusters : public eicrecon::JFactory
         m_input_tag = "LFHCALRecHits";
 
         m_splitCluster=false;
-        m_minClusterHitEdep=1 * dd4hep::MeV;
-        m_minClusterCenterEdep=100.0 * dd4hep::MeV;
+        m_minClusterHitEdep=1 * unit::MeV;
+        m_minClusterCenterEdep=100.0 * unit::MeV;
 
         // neighbour checking distances
         m_sectorDist=0 * dd4hep::cm;
```

Comment:
```suggestion
        m_sectorDist=0 * unit::cm;
```

Reply:
Same situation as with timing. The original values with units may be subtly broken (unit systems differ by a factor of 10, not 10^9 like for timing) for non-zero. Putting the EDM4hep units should fix that, but I'd like to review that more carefully, and, perhaps, separately.

Reply:
Good point.. probably not a good idea to update _everything_ in this PR.